File: 백준/Bronze/10708. 크리스마스 파티/크리스마스 파티.java
import java.io.*;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        StringBuilder sb = new StringBuilder();
        int N = Integer.parseInt(br.readLine().trim());
        int M = Integer.parseInt(br.readLine().trim());
        String[] answerStr = br.readLine().split(" ");
        int[] answer = new int[M];
        int[] score = new int[N];
        for (int i = 0; i < M; i++) {
            int extra = 0;
            answer[i] = Integer.parseInt(answerStr[i]);
            String[] targetsStr = br.readLine().split(" ");
            int[] targets = new int[N];
            for (int j = 0; j < N; j++) {
                targets[j] = Integer.parseInt(targetsStr[j]);
                if (targets[j] == answer[i]) {
                    score[j] += 1;
                } else {
                    extra += 1;
                }
            }
            score[answer[i] - 1] += extra;
        }
        br.close();
        for (int num : score) {
            sb.append(num).append('\n');
        }
        System.out.print(sb.toString());
    }
}

File: 백준/Bronze/10708. 크리스마스 파티/크리스마스 파티.py
import sys
input = sys.stdin.readline

N = int(input().strip())
M = int(input().strip())

answers = list(map(int, input().split()))
score = [0] * N

for i in range(M):
    targets = list(map(int, input().split()))
    extra = 0
    for j in range(N):
        if targets[j] == answers[i]:
            score[j] += 1
        else:
            extra += 1
    score[answers[i] - 1] += extra

print("\n".join(map(str, score)), end="")